Atomic/backends/_docker.py

```diff
@@ -399,11 +399,18 @@ def uninstall(self, iobject, name=None, **kwargs):
         if cmd:
             return util.check_call(cmd, env=atomic.cmd_env())
 
+        install_data = util.InstallData.get_install_data_by_id(iobject.id)
+        system_package_nvra = install_data.get("system_package_nvra", None)
+        if system_package_nvra:
+            if os.path.exists("/usr/bin/dnf"):
+                util.check_call(["dnf", "remove", "-y", system_package_nvra])
+            else:
+                util.check_call(["yum", "remove", "-y", system_package_nvra])
```

Comment:
this code is almost the same in `syscontainers.py`.  We should refactor this to `rpm_host_install.py` and make both Docker and system containers use the same functions.